.github/workflows/target_workflow.yml

```diff
@@ -0,0 +1,91 @@
+name: QC Target Workflow
+
+on:
+  repository_dispatch:
+    types: [trigger-workflow]
+
+permissions:
+  id-token: write
+  contents: read
+  packages: write
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    environment: main
+    defaults:
+        run:
+            shell: bash
+            working-directory: ./terraform/aws/implementation
+    outputs:
+            tf_env: ${{ steps.set-environment.outputs.tf_env }}
+    steps:
+      - name: Checkout code 
```

Comment:
I'm confused why we need to checkout code from `dibbs-query-connector` if the service name and version are passed in separately.